webpack.config.js

```diff
@@ -65,11 +65,27 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader'
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
```

Comment:
Are there any other changes that need to be made before we can switch this to `^\/client/(.)*\/index.jsx$`?